src/book.rs

```diff
@@ -273,6 +270,16 @@ impl Book {
         }
     }
 
+    fn prune(&mut self) {
```

Comment:
seems like this is a less efficient way to do this than before? Doesn't this mean after each trade we need to iterate over the entire orderbook?

Reply:
> seems like this is a less efficient way to do this than before?

To be clear, this wasn't happening before.

> Doesn't this mean after each trade we need to iterate over the entire orderbook?

Iteration occurs over each **price-level**, not each order. Although the "entire" book will be touched.

Reply:
sweet sounds good @sporejack